internal/common/settings.go

```diff
@@ -122,12 +122,12 @@ func readSourceOfTrustSettings() (*SourceOfTrustSettings, error) {
 	return sourceOfTrust, nil
 }
 
-func readFileSourceOfTrustSettings() (*FileSourceOfTrustSettings, error) {
+func readFileSourceOfTrustSettings() (*FileTrustSourceSettings, error) {
 	if !viper.IsSet("trustsource.file.domains") {
 		return nil, errors.New("trustsource.file.domains is required but not found")
 	}
 
-	fileSettings := new(FileSourceOfTrustSettings)
+	fileSettings := new(FileTrustSourceSettings)
```

Comment:
Why not call `NewFileTrustSource` here?

Reply:
`FileTrustSourceSettings` is a container for the settings in the `trustsource.file` block of the plugin settings file while `FileTrustSource` is the class that actually reads and parses the trust sources. These two have different purposes and while a case could be made that the the intermediary `FileTrustSourceSettings` between the settings file and creation of a `FileTrustSource` instance is unnecessary I think that discussion is outside the scope of this PR.

Reply:
🤦‍♂ I should read before typing. This makes sense and I'm okay with it